translate/misc/optrecurse.py

```diff
@@ -98,7 +98,8 @@ def format_option_strings(self, option):
 
 
 class StdoutWrapper:
-    out = sys.stdout
```

Comment:
This is to be able to capture the output from a unit test. Without the constructor, the assignment is done too early and no output is captured.